engine.py

```diff
@@ -313,23 +487,31 @@ def get_args():
     args = parser.parse_args()
     return args
 
-
 def main(args):
     """
     Kicks off the engine with the given arguments.
     """
+
     try:
+        if args["strategy"]:
+            strategy = args["strategy"]
+        elif args["strategy_index"]:
+            strategy = LIBRARY[int(args["strategy_index"])][0]
+        else:
+            # Default to first strategy
+            strategy = LIBRARY[0][0]
```

Comment:
can we have this default to a different strat? strat 0 isn't a good general purpose one. can we do `    ("[TCP:flags:PA]-duplicate(tamper{TCP:load:corrupt}(tamper{TCP:chksum:corrupt},),)-|",  80,  100,  0),` (line 6, 5th strat in library)? this one works in more places and doesn't rely on the outbound packet